core/workspace_svg.js

```diff
@@ -1255,17 +1318,19 @@ Blockly.WorkspaceSvg.prototype.onMouseWheel_ = function(e) {
  */
 Blockly.WorkspaceSvg.prototype.getBlocksBoundingBox = function() {
   var topBlocks = this.getTopBlocks(false);
+  var topComments = this.getTopComments(false);
```

Comment:
The bounding box now has more than blocks.

The name of this function may eventually need to change, but probably not because it's public.  It's really "get the bounding box of things in the workspace that impact the size of the workspace".